apis/core/v1alpha1/annotations.go

```diff
@@ -35,6 +35,15 @@ const (
 	// TODO(jaypipes): Link to documentation on cross-account resource
 	// management
 	AnnotationOwnerAccountID = AnnotationPrefix + "owner-account-id"
+	// AnnotationTeamID is an annotation whose value is the identifier
+	// for the AWS team ID to manage the resources.  If this annotation
+	// is set on a CR, the Kubernetes user is indicating that the ACK service
+	// controller should create/patch/delete the resource in the specified AWS
+	// role for this team ID. In order for this cross-account resource management
+	// to succeed, the AWS IAM Role that the ACK service controller runs as needs
+	// to have the ability to call the AWS STS::AssumeRole API call and assume an
+	// IAM Role in the target AWS Account.
+	AnnotationTeamID = AnnotationPrefix + "team-id"
```

Comment:
Thoughts: the more I think about this, the more I think maybe this should just be a `role-arn` .. looks like every team-id is just a RoleARN at the end of the day?

Reply:
>but I assume it makes it easier for the user when reasoning who should have which permissions? 🤔

You convinced me with this one.. it makes sense to reason that a team have a role...